source/_posts/azure-purview.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Azure Purview
+date: 2021-03-22 19:38:02
+tags:
+---
+
+If you don't know what kind of data you have and where it resides, it's difficult to use it. Azure Purview can help with that! 
+
+Azure Purview is a unified data governance service that helps you scan and map all your data no matter where the data is stored! tThis data could be on-premise, multi-cloud, or SaaS data.
+
+Azure Purview is an update to the existing Azure Data Catalog Gen 2 service. Azure Data catalog Gen 2 focused more on managing metadata instead of real data governance and was capable of mapping, searching for, and tagging data within data sources but wasn't able to classify its data.This made it hard to be compliant to different laws pertaining data protection, like GDPR. 
+
+Azure Purview allows you to answer the following questions:
+
+- What kind of data do I have?
+- What are all the different tables and fields in the database? 
+- Where is any sensitive data? 
+
+# How does Azure Purview work?
+
+It all starts with the data. Organisations can have many different data assets like files, tables,reports and databases. These data sources can exist across different environments, like in the public cloud, on-premise, and SaaS environments. Azure Purview even supports multi-cloud. Because of this support, you do not have to move data from on-premise or other cloud providers to Azure: everything you want to scan can stay where it is. 
+
+<img src="/images/azure-purview/cloud-model.png" />
+
+You can then connect these different environments with Azure Purview via the different connectors that are available. 
+
+Purview can scan these data sources to extract different things like metadata, lineage and data classification. The scans operate serverless, so you only pay for what you use. 
+
+When the scanning is done, all the data that is found is published to the Azure Purview datamap. The data map is like a graph describing all the data and relationships between data assets across different environments.The more data you scan, the more powerful and useful this graph becomes!
```

Comment:
Choose datamap, data map or Data Map (as per https://azure.microsoft.com/en-us/services/purview/), and be consistent! Personally, I'd follow the product page.

 environments.The >>  environments. The (whitespace)
 
 again check environments vs. platforms (do this for the entire document, I will not point out the other instances from here on)